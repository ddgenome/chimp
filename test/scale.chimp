## scaling test
mechanism "scale.mech"
## parameter input
parameter "scale.par"
## scale test task
task "scale.task"
