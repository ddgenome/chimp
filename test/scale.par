# parameter input for scaling test
A1f	2.0e1	# mol/m^3/s
E1f	3.2e4	# J/mol
A1r	1.0e1	# mol/m^3/s
E1r	3.0e4	# J/mol
A2f	5.0e0	# mol/m^3/s
E2f	2.8e4	# J/mol
A2r	4.9e0	# mol/m^3/s
E2r	3.0e4	# J/mol
