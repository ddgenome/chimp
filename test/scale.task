# -*- text -*-
# scaling test task input
begin model scale_19
  output "scale.out"
  begin integrator kmc
    scale 1.0e19
    begin state
      begin quantity
	c[A] = 1.0e0		# mol/m^3
	c[B] = 1.0e0		# mol/m^3
      end quantity
      begin output
	(1.0e1 1.0e2 1.0e1)
      end output
      begin reactor batch
	temperature 4.2e2	# K
	pressure 1.0e5		# Pa
	volume 1.0e1		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity concentration
      end reactor
    end state
  end integrator
end model

begin model scale_18
  output "scale.out"
  begin integrator kmc
    scale 1.0e18
    begin state
      begin quantity
	c[A] = 1.0e0		# mol/m^3
	c[B] = 1.0e0		# mol/m^3
      end quantity
      begin output
	(1.0e1 1.0e2 1.0e1)
      end output
      begin reactor batch
	temperature 4.2e2	# K
	pressure 1.0e5		# Pa
	volume 1.0e1		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity concentration
      end reactor
    end state
  end integrator
end model

#  begin model scale_17
#    output "scale.out"
#    begin integrator kmc
#      scale 1.0e17
#      begin state
#        begin quantity
#  	c[A] = 1.0e0		# mol/m^3
#  	c[B] = 1.0e0		# mol/m^3
#        end quantity
#        begin output
#  	(1.0e1 1.0e2 1.0e1)
#        end output
#        begin reactor batch
#  	temperature 4.2e2	# K
#  	pressure 1.0e5		# Pa
#  	volume 1.0e1		# m^3
#  	rate_numerator moles
#  	rate_denominator volume
#  	fluid_quantity concentration
#        end reactor
#      end state
#    end integrator
#  end model

begin model scale_18v1
  output "scale.out"
  begin integrator kmc
    scale 1.0e18
    begin state
      begin quantity
	c[A] = 1.0e0		# mol/m^3
	c[B] = 1.0e0		# mol/m^3
      end quantity
      begin output
	(1.0e1 1.0e2 1.0e1)
      end output
      begin reactor batch
	temperature 4.2e2	# K
	pressure 1.0e5		# Pa
	volume 1.0e0		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity concentration
      end reactor
    end state
  end integrator
end model
