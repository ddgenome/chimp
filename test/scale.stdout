chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task scale.par... completed; 0u 0s 0w
chimp: performing task scale_19... completed; 0.65u 0s 0.65w
chimp: performing task scale_18... completed; 6.42u 0s 6.42w
chimp: performing task scale_18v1... completed; 0.65u 0s 0.65w
